src/components/ui/Select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Select as UISelect } from '@faststore/ui'
+import type { SelectProps } from '@faststore/ui'
+import { CaretDown as CaretDownIcon } from 'phosphor-react'
+
+import './select.scss'
+
+type SelectOptions = {
+  [key: string]: string
+}
+
+interface UISelectProps extends SelectProps {
+  /*
+   * Defines the options available in the select. The SelectOptions object
+   * keys are the property names, while the values correspond  to the text that
+   * will be displayed in the UI
+   */
+  options: SelectOptions
+  /*
+   * Specifies the text that will be displayed in the label right next to the Select.
+   * If omitted, the label will not be rendered.
+   */
+  labelText?: string
+}
+
+export default function Select({
+  options,
+  onChange,
+  labelText,
+  value,
+  'aria-label': ariaLabel,
+  className,
+  testId,
+}: UISelectProps) {
+  return (
+    <div data-ui-select className={className}>
+      {labelText && <label htmlFor="ui-select">{labelText}</label>}
+      <UISelect
+        data-testid={testId}
+        onChange={onChange}
+        value={value}
+        aria-label={ariaLabel}
+        id="ui-select"
```

Comment:
Having this ID hardcoded here is going to be a problem if multiple instances of this component are rendered on the same page because we would have non-unique IDs for different `select`s.

Maybe we should allow an `id` prop so different IDs can be specified?

Reply:
Hmm, not sure. This `id` sole purpose is to link it with the `label`, right? I think there's not much room for modification there.

Styling can be done by the `data-attributes`, and classes can be added do the wrapping `div`. Do you think of any scenario/use case that adding this "flexibility" to add a custom `id` might be useful?

Reply:
> This id sole purpose is to link it with the label, right? 

Right! 

I'm talking about the scenario of someone using this component multiple times on the same, e.g. to render different `select`s. They would all be rendered with the same `id="ui-select"` attribute, which is a problem because no `id`s should be repeated. This would cause a bug such as clicking on the 2nd `select`'s `label` activating the 1st `select`.
https://codepen.io/voahhsllbutvrxyoje/pen/podbJGa
![CleanShot 2022-02-02 at 15 32 33](https://user-images.githubusercontent.com/381395/152216108-151cdfdf-8966-4bae-a57a-b57392b1f115.gif)

Reply:
Please consider what Filipe said. We faced this problem with the `Accodion`. I think for now the best approach is allow an `id` or could be an id prefixe.

Reply:
Makes total sense. Thanks people, I'll work on it.

Reply:
Done at https://github.com/vtex-sites/base.store/pull/299/commits/40a8fd4988be2ddedd3e93d4502e27f743d2f9ed

Reply:
Hey, what's the problem with this hardcoded id? I have no context about it.